app/src/main/java/me/saket/dank/ui/subreddit/UserProfileSheetView.java

```diff
@@ -200,32 +202,11 @@ void onClickComments() {
   void onClickSubmissions() {
   }
 
-  @OnClick(R.id.userprofilesheet_logout)
-  void onClickLogout(TextView logoutButton) {
-    if (confirmLogoutTimer.isDisposed()) {
-      logoutButton.setText(R.string.userprofile_confirm_logout);
-      confirmLogoutTimer = Observable.timer(5, TimeUnit.SECONDS)
```

Comment:
You should also remove the `confirmLogoutTimer` and `logoutDisposable` fields as they are no longer used.